main.go

```diff
@@ -68,13 +68,11 @@ func main() {
 	}
 
 	if flags.dry == false {
-		resolve := output.Stdout("Attempting to push new tag to GitHub...")
+		output.Stdout("Attempting to push new tag to GitHub...")
```

Comment:
I'm assuming that you would still provide the `--output-tag` option (which makes it so `output.Stdout` does not actually print anything). So if you do just `semantics` it will print out that message but if you do `semantics --output-tag` it will only output the tag.

I only did that to make minimal changes to the code though. I could see myself also liking it if we get rid of the `--dry-run` and `--output-tag` options and have outputting the tag on success be the default.